### PR TITLE
home page

### PR DESCRIPTION
**Pull request by prakashsakari**
1. Created Landing page
2. Added Navigation Bar
3. Category, Banner and Footer
hm-1
r
hm-2

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            href="https://github.com/prakashsakari/"
+            class="link"
+            target="_blank"
+            rel="noreferrer"
+          >
+            <img
+              class="source-image"
+              src="https://therightfit.netlify.app/assets/github-logo.png"
+              alt="GitHub"
+            />
```
Comment (rohanmathur91) [inline src/components/Footer/Footer.js]: Make an array for your socials data and loop through it

File: src/pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import {
+    Navbar,
+    Banner,
+    Category,
+    NewArrivals,
+    Footer
+  } from "../components/index";
+
```
Comment (rohanmathur91) [inline src/pages/Home.js]: No need to mention /index

**Review by rohanmathur91 — APPROVED** [review]
- Good use of semantic HTML
 - Exports for components are done nicely
 - Home page layout is nice, since you only put the components in that page, and your page is ready.

File: src/components/Footer/Footer.js (hunk 1 above)
Comment (prakashsakari) [reply src/components/Footer/Footer.js]: updated this, thanks

File: src/components/Banner/Banner.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { Link } from "react-router-dom";
+const Banner = () => {
+  return (
+    <main class="main-page-container">
+      <section class="main-banner">
+        <Link to="/" class="link">
```
Comment (Aditi0505) [inline src/components/Banner/Banner.js]: Suggestion: Please change class to className

File: src/components/Category/Category.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+              <Link class="link relative effect" to="/">
+                <div class="category category-men">
+                  <img
+                    class="category-image border-radius-4"
+                    src={imgUrl}
+                    alt={alt}
+                  />
+                </div>
+                <div class="overlay d-flex direction-column absolute top-0 left-0 d-flex justify-center align-center text-bg">
+                  <div class="men-overlay d-flex justify-center">
```
Comment (Aditi0505) [inline src/components/Category/Category.js]: Please change class to className at all places.

File: src/pages/Home.js (hunk 2 above)
Comment (prakashsakari) [reply src/pages/Home.js]: done and updated, thanks

File: src/components/Footer/Footer.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+          </a>
+          <a
+            href="https://twitter.com/prakashsakari"
+            class="link"
+            target="_blank"
+            rel="noreferrer"
+          >
+            <img
```
Comment (ishitakeshawani) [inline src/components/Footer/Footer.js]: Why did you use images instead of icons?

File: src/index.css [hunk 6]
```diff
@@ -0,0 +1,13 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
```
Comment (Aditi0505) [inline src/index.css:7]: Please let me know the purpose of setting these? Is it due to some extension?

File: src/staticData/categories.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
+const categories = [
```
Comment (ishitakeshawani) [inline src/staticData/categories.js]: You can add these categories in DB folder.

**Review by ishitakeshawani — APPROVED** [review]
Added a suggestion.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestion.
Another suggestion would be to create a template using mockbee for easier api integration.

File: src/components/Footer/Footer.js (hunk 5 above)
Comment (prakashsakari) [reply src/components/Footer/Footer.js]: I did it to make it fit for any shape of image, be it rounded or square

File: src/staticData/categories.js (hunk 7 above)
Comment (prakashsakari) [reply src/staticData/categories.js]: Ok will do that.

File: src/index.css (hunk 6 above)
Comment (prakashsakari) [reply src/index.css:7]: These are default styling created inside index.css when we create a react app

File: src/components/Category/Category.js (hunk 4 above)
Comment (prakashsakari) [reply src/components/Category/Category.js]: updated, thanks aditi

File: src/components/Banner/Banner.js (hunk 3 above)
Comment (prakashsakari) [reply src/components/Banner/Banner.js]: updated, thanks aditi

**Pull request merged**